src/main/java/org/junitpioneer/jupiter/NullEnum.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2016-2021 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+/**
+ * Dummy enum class used as default value for optional attributes of
+ * annotations.
+ *
+ * @since 1.4.0
```

Comment:
Is the version number correct? I'm not sure how to stay in sync with Shipkit.

Reply:
> Is the version number correct? I'm not sure how to stay in sync with Shipkit.

I personally don't like this `@since`, because when you develop and open a PR you don't know when this will get merged / shiped.

Reply:
I think it's mostly useful for the users to understand when something appeared in the library without going to the library repository. Of course, either the author or who is merging the PR must ensure that the version is correct.

I'm fine with any solution, just let me know what's the preferred style about it.

Reply:
I agree with Bukama, just remove the `@since` for now.